src/planscape/forsys/get_forsys_inputs.py

```diff
@@ -197,6 +225,11 @@ def get_condition_header(self, condition: str) -> str:
 
 
 class ForsysProjectAreaRankingInput():
+    # Treatment cost per kilometer-squared (in USD)
+    # TODO: make this variable based on a user input and/or a treatment cost
+    # raster.
+    TREATMENT_COST_PER_KM_SQUARED = 5000 * 1000 * 1000
```

Comment:
@mskim3494 

Is 5000 USD per km^2 reasonable as a treatment cost constant to start?